.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Status
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
drop 10.x it is out of maintenance.

Reply:
Both of your comments should now be addressed :)